golang/example/example_lambda_args.go

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020 - present Kurtosis Technologies LLC.
+ * All Rights Reserved.
+ */
+
+package example
+
+type ExampleLambdaArgs struct {
+	// Indicates that this Lambda is being run as part of CI testing in Kurtosis Core
+	IsKurtosisCoreDevMode bool `json:"isKurtosisCoreDevMode"`
+	// Indicates the log level for this Lambda implementation
+	logLevel string `json:"logLevel"`
```

Comment:
think this should be capitalized (from my understanding, all JSON fields need to be capitalized-public)

Reply:
Yes, you are right! I forgot to do it, my mistake, good catch!